lib/users.ts

```diff
@@ -0,0 +1,33 @@
+import { prisma } from "@/db/db";
+
+export type User = {
+  id: string;
+  clerkId: string;
+  email: string | null;
+};
+
+export async function getUsers() {
+  return await prisma.user.findMany({
+    orderBy: { id: "asc" },
+  });
+}
+
+export async function getOneUser(id: string) {
+  return await prisma.user.findUnique({
+    where: { id },
+  });
+}
+
+export async function checkClerkId(clerkId: string) {
```

Comment:
I would rename this method because here we are not checking anything, what we are doing is retrieving a user by clerkId.

```suggestion
export async function getUserByClerkId(clerkId: string) {
```